Templates/FinniversKitView.xctemplate/___FILEBASENAME___.swift

```diff
@@ -18,14 +18,21 @@ class ___FILEBASENAMEASIDENTIFIER___: <#View Class#> {
         setup()
     }
 
-    private func setup() {
+    // MARK: - Superclass Overrides
+    
+}
+
+// MARK: - Private
+
+private extension ___FILEBASENAMEASIDENTIFIER___ {
+    func setup() {
```

Comment:
This needs to be private.

Reply:
Never mind. Marking it private will render it inaccessible to the init methods. Not so sure about putting this in the private extension anymore. Should probably stay in the `Setup` section?

Reply:
Yes it's innherited from the extension, so there is no need to explicitly use `private` 😊

Reply:
I think it would be nice to keep it in the `private extension` 🤔 I think its nice to keep the original class scope free of self defined methods and only have super class overrides there.

Reply:
Also, setup code is long and boring so then you wouldn't have to scroll trough it as much 😂

Reply:
Any thought @3lvis ?

Reply:
My initial thought was that the view setup (constraints etc) will be far away from the properties holding the views. Not sure if this is a big problem though. I agree that all methods created by us should stay in the public/private section.